terminus-ui/src/icon/icon.component.md

```diff
@@ -75,10 +75,14 @@ Custom icons are accessed via a `ts-icon` `@Input`:
 
 <!-- Access custom icon -->
 <ts-icon svgIcon="csv"></ts-icon>
+<ts-icon svgIcon="engage"></ts-icon>
+<ts-icon svgIcon="lightbulb"></ts-icon>
```

Comment:
We don't need to add all custom options here since they are listed just below in the table